CMakeLists.txt

```diff
@@ -1,29 +1,11 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 2.8.8)
```

Comment:
as usual it's too painful to upgrade Travis' environment - I'll get rid of this CMake version upgrade requirement